pkgs/development/python-modules/pysmartapp/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+, httpsig
+}:
+
+buildPythonPackage rec {
+  pname = "pysmartapp";
+  version = "0.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1i5f6as4f0jnba2zynni0v0y9fkf2qfavpyz73nk78as4j8ysnr9";
+  };
+
+  propagatedBuildInputs = [
+    httpsig
+  ];
+
+  pythonImportsCheck = [ "pysmartapp" ];
```

Comment:
Without `doCheck = false;` the build process fails.

```bash
Check whether the following modules can be imported: pysmartapp
setuptoolsCheckPhase
Executing setuptoolsCheckPhase
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
WARNING: The directory '/homeless-shelter/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
[...]
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff5ef5640>: Failed to establish a new connection: [Errno -2] Name or service not known')': /simple/tox/
ERROR: Could not find a version that satisfies the requirement tox<4.0.0,>=3.5.0
ERROR: No matching distribution found for tox<4.0.0,>=3.5.0
error: Command '['/nix/store/d44wd6n98f93hjr6q1d1phhh1hw7a17d-python3-3.8.8/bin/python3.8', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/build/tmpwigins6y', '--quiet', 'tox<4.0.0,>=3.5.0']' returned non-zero exit status 1.
builder for '/nix/store/nxy7d2bk1i0q7r0cy5033sdgdxzm6c65-python3.8-pysmartapp-0.3.3.drv' failed with exit code 1
error: build of '/nix/store/nxy7d2bk1i0q7r0cy5033sdgdxzm6c65-python3.8-pysmartapp-0.3.3.drv' failed
```

But the upstream repo contains tests. Switch to the source from GitHub (see above) and add the test requirements as `checkInputs` and as inputs.

```suggestion
  checkInputs = [
    pytest-asyncio
    pytestCheckHook
  ];

  pythonImportsCheck = [ "pysmartapp" ];
```